boot.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"github.com/deis/registry-token-refresher/pkg"
+	"github.com/deis/registry-token-refresher/pkg/credentials"
+	"k8s.io/kubernetes/pkg/api"
+	kcl "k8s.io/kubernetes/pkg/client/unversioned"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+)
+
+var appList []string
+var registryLocation = os.Getenv("DEIS_REGISTRY_LOCATION")
+
+func getDiff(namespaceList []api.Namespace) []string {
+	var added []string
+	var apps []string
```

Comment:
you should pre-allocate size for this slice. `apps := make([]string, len(namespaceList))`